src/pkg/api/handlers.go

```diff
@@ -935,3 +936,22 @@ func healthz(w http.ResponseWriter, _ *http.Request) {
 		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
 	}
 }
+
+// @Description get list of clusters with context names from local kubeconfig
+// @Tags clusters
+// @Produce json
+// @Success 200
+// @Router /api/v1/clusters [get]
+func getClusters(ks *session.K8sSession) func(w http.ResponseWriter, r *http.Request) {
+	return client.ServeClusters(ks.InCluster, ks.CurrentCtx)
+}
+
+// @Description Swith the currently viewed cluster
```

Comment:
```suggestion
// @Description Switch the currently viewed cluster
```

I think this is the only one you need to actually accept, the others are generated?

Reply:
Update: I deleted my other suggestions as this is likely the only one needed